.github/workflows/update_app_templates_hash.yml

```diff
@@ -0,0 +1,43 @@
+name: Update aws/aws-sam-cli with latest commit hash from aws/aws-sam-cli-app-templates
+
+on:
+  schedule:
+    - cron: "0 * * * *"
```

Comment:
Can you add a comment on what this cron is? I hate trying to read cron.

Reply:
You are right, this is for every hour, let me add that as a comment